crates/apub/src/protocol/objects/page.rs

```diff
@@ -52,6 +52,7 @@ pub struct Page {
   #[serde(deserialize_with = "deserialize_one_or_many", default)]
   pub(crate) cc: Vec<Url>,
   pub(crate) content: Option<String>,
+  pub(crate) alt_text: Option<String>,
```

Comment:
I looked in the [activitypub vocab](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-image), but it only seems to support them as  `name` on images. Not sure what the best way to do this is.

Reply:
You should put it into the [Attachment field](https://github.com/LemmyNet/lemmy/blob/main/crates/apub/src/protocol/objects/page.rs#L81) as `name`, thats also what Mastodon uses.

Example: `curl -H "Accept: application/activity+json" https://twit.social/@glennf/112004543167309282 | jq`

Reply:
Done.